src/zfs-auto-snapshot.sh

```diff
@@ -186,7 +191,7 @@ do_snapshots () # properties, flags, snapname, oldglob, [targets...]
 					fi
 				fi
 			fi
-		done
+		done <<< "$SNAPSHOTS_OLD"
```

Comment:
I've merged @leecallen as per instructions on the wiki and I've changed to /bin/bash but now I'm getting echoed the error "The filesystem argument list is empty." (line 350 is the print)
I'm on Debian 8.8